lib/modules/apostrophe-browser-utils/public/js/lean.js

```diff
@@ -445,4 +440,19 @@
     return path + '.' + file.extension;
   };
 
+  // Returns true if the uri references the same site (same host and port) as the
+  // current page. Cross-browser implementation, valid at least back to IE11.
+  // Regarding port numbers, this will match as long as the URIs are consistent
+  // about not explicitly specifying the port number when it is 80 (HTTP) or 443 (HTTPS),
+  // which is generally the case.
+
+  apos.utils.sameSite = function(uri) {
```

Comment:
We should have a test for this, right?

Reply:
It's frontend code, so not easily tested in our current harness, but I tested it at the console with several distinct cases in IE11 and in Chrome.

Reply:
Oh true. Sounds good.